trace-protobuf/build.rs

```diff
@@ -0,0 +1,76 @@
+use std::io::Result;
```

Comment:
this build script was added to re-compile `.proto` files we use, adding `#[serde(default)]` annotations to certain fields.

this allows us to directly de-serialize spans into these proto generated structs, as opposed to first de-serializing to a struct with Option<> fields, then converting to the proto generated Span struct (using unwrap_or_default on Option<> fields)